### DISCUSSION TITLE
DIA-NN output - CV's graphs

### DISCUSSION THREAD

**Discussion by pejota66** [vdemichev/DiaNN#704] [Q&A]
Hello,

I am able to generate and obtain the report PDF resulting from DIA-NN.
Although in some cases I do obtain a graph corresponding to "Protein groups, 1% FDR highlighting the CVs (10%, 20%)" in other cases, these types of graphs are not present. 
How should I proceed to fix this and always obtain the CV's graphs or How can I do to calculate it?

Many thanks in advance.

**Comment by vdemichev** [comment]
I would suggest to generate all plots needed with custom scripts from the output .tsv files, it's more reliable that using PDF report. The latter is purely for convenience when testing the system and not for publication-quality work. Specifically with CVs, naming samples like [name]_replicate will work, e.g. A_1, A_2, A_3, B_1, B_2, B_3. Otherwise CVs are not calculated.

Best,
Vadim

**Reply by pejota66** [reply]
Many thanks, Vadim.
Do you recommend using any specific scripts/packages, for example, DiaNN R Package?

Best

**Reply by vdemichev** [reply]
This is very simple to write in vanilla R
